src/components/WebexVideoSettings/WebexVideoSettings.scss

```diff
@@ -0,0 +1,26 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}-video-settings {
+  position: relative;
+  .camera-warning {
+    color: $md-red-60;
+    display: flex;
+    justify-content: space-between;
+    align-items: start;
+    padding: 0.25rem 0 0.25rem 0;
+
+    .message {
+      margin-left: 0.625rem;
+      font-size: 1rem;
+    }
+  }
+  .preview {
+    position: absolute;
```

Comment:
I don't see a reason for absolutely positioning this element. If the container is a normal div, or a column flex container, the preview should be displayed fine under the control.

Reply:
ok, I see now, this is just the "Preview" banner, can you display it relative to the LocalMedia component?